spec/models/user_spec.rb

```diff
@@ -112,4 +112,21 @@
       user.ac?.should eq(true)
     end
   end
+
+  describe '#all_emails' do
+    it 'contains primary email' do
+      user.all_emails.should include(user.email)
+    end
+    it 'contains confirmed additional emails' do
+      user = create(:user)
+      additional_email = create(:additional_email, user: user, confirmed_at: Date.today)
+      user.all_emails.should include(additional_email.email)
+    end
+
+    it 'does not contain unconfirmed additional emails' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.